src/main.ts

```diff
@@ -85,12 +66,11 @@ async function setDockleEnvVariables(): Promise<{ [key: string]: string }> {
         dockleEnv["DOCKLE_PASSWORD"] = password;
     }
 
-    dockleEnv["DOCKLE_EXIT_CODE"] = "5";
-
     return dockleEnv;
 }
 
 async function runTrivy(): Promise<number> {
+    whitelistHandler.init();
```

Comment:
This initialises both Trivy and Dockle whitelists. Move this call to `run`.

Reply:
OK